crates/sui-core/src/authority_aggregator.rs

```diff
@@ -180,6 +169,12 @@ impl AuthAggMetrics {
                 registry,
             )
             .unwrap(),
+            cert_broadcasting_post_quorum_timeout: register_int_counter_with_registry!(
```

Comment:
This probably happens for every txn. Maybe count total  remaining_tasks at timeout instead?

Reply:
I'm not sure if this happens that often, but counting that is a good idea, will add.